src/Core/Content/ScaleUnit/Extension/ProductExtension.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\ScaleUnit\Extension;
+
+use Shopware\Core\Content\Product\ProductDefinition;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityExtension;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\CascadeDelete;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\ManyToManyAssociationField;
+use Shopware\Core\Framework\DataAbstractionLayer\FieldCollection;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[Package('inventory')]
+class ProductExtension extends EntityExtension
```

Comment:
why using an extension, if we directly add it to the core? just change the product directly